src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -30,11 +34,15 @@ class JsonAdaptedPerson {
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
                              @JsonProperty("email") String email, @JsonProperty("address") String address,
+                             @JsonProperty("money_owed") Integer moneyOwed,
+                             @JsonProperty("monet_paid") Integer moneyPaid,
```

Comment:
think there is typo error here. should be "money_paid".

Reply:
Technically, this is not a typo: `@JsonProperty("money_owed")` specifies the property name in JSON file and maps the value from JSON into the corresponding variable (in this case, `moneyOwed` in the parameter). Therefore, the program here works as intended. You can read more from this [post](https://stackoverflow.com/questions/12583638/when-is-the-jsonproperty-property-used-and-what-is-it-used-for) if interested. 

But it is indeed a good point to standardize the naming convention to avoid potential mismatches in future. Therefore, this is addressed in [commit dab5c94](https://github.com/AY2223S1-CS2103T-T09-4/tp/pull/93/commits/dab5c94f46889e2a57e36053efeefb05bcddab06) and the format of JSONProperty is standardized to be camelCase.

Reply:
Correction for my earlier comment: in line 38 there is a typo and has been fixed in [commit b46e1b4](https://github.com/AY2223S1-CS2103T-T09-4/tp/pull/93#:~:text=Fix%20the%20typo-,b46e1b4,-Merge%20state)